website/users/models.py

```diff
@@ -47,65 +46,42 @@ def create_superuser(self, username, **kwargs):
         return self._create_user(username, **kwargs)
 
 
-class User(BaseUser):
+class User(AbstractUser):
     """User object."""
 
     objects = UserManager()
 
     USERNAME_FIELD = "username"
 
-    REQUIRED_FIELDS = []
+    REQUIRED_FIELDS = ["email"]
 
-    class Meta:
-        """Meta class for Users."""
+    username = models.CharField(max_length=8, unique=True)
+    email = models.EmailField(unique=True)
+    full_name = models.CharField(max_length=100)
```

Comment:
Why do we have `first_name`, `last_name` and `full_name`? Can't we remove either one of these?